### DISCUSSION TITLE
WMATAUI.swift

### DISCUSSION THREAD

**Discussion by emma-k-alexandra** [emma-k-alexandra/WMATA.swift#26] [General]
Hey @rhwood,

Noticed the [WMATAUI.swift](https://github.com/rhwood/WMATAUI.swift) project you've got going. Looks pretty cool.

If there's anything I can do to contribute, let me know. I have a bunch of UI and SwiftUI related utilities built on top of this project for the app I publish, Should I Run. Even stuff for watchOS. I could look into them to see if there's anything useful.

**Comment by emma-k-alexandra** [comment]
At the very least I will probably start trying to use your package for color stuff, which looks way more in depth than the research I did myself!

**Comment by rhwood** [comment]
@emma-k-alexandra anything you think anyone can benefit from would be appreciated. I'm trying to avoid dependencies on things other than SwiftUI (although I clearly failed with the font metrics problem of attempting to use the WMATA font).
